pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -344,6 +349,37 @@ func (b *BridgePodInterface) createBridge() error {
 	return nil
 }
 
+func (b *BridgePodInterface) switchPodInterfaceWithDummy() error {
+	originalPodInterfaceName := b.podInterfaceName
+	newPodInterfaceName := fmt.Sprintf("%s-nic", originalPodInterfaceName)
```

Comment:
This is probably a corner case, but there is a limit as to the length of interfaces in Linux. If you append four more characters, in a degenerate case you may end up with a name that is not accepted by the kernel. Are there any standards in Kubernetes as to the maximal length of the pod interface name?